jenkins/jobs/image-pld.yaml

```diff
@@ -1,6 +1,5 @@
 - job:
-    name: "image-pld"
-    concurrent: false
+    name: "image-pld"    concurrent: false
```

Comment:
Still not quite right, you're missing a line break here :)

Reply:
Now the diff looks good.